test/server/cards/01_SOR/units/SeventhSisterImplacableInquisitor.spec.ts

```diff
@@ -0,0 +1,42 @@
+describe('Seventh Sister', function () {
+    integration(function (contextRef) {
+        describe('Seventh Sister\'s triggered ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['battlefield-marine', 'seventh-sister#implacable-inquisitor']
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                        spaceArena: ['cartel-spacer'],
+                        leader: { card: 'boba-fett#daimyo', deployed: true },
+                        base: 'chopper-base'
+                    },
+                });
+            });
+
+            it('may deal 3 damage to a ground unit if attacked base', function () {
```

Comment:
Could combine the two tests into one using a reset method to reset state between attacks (see the Rukh tests for an example). Can't remember if I said this to you already or not, but we have to consolidate tests a bit (even though it's bad practice) because the tests are slow and we have limited Github Actions time at the free tier